src/main/java/com/dev/cinema/model/dto/MovieSessionResponseDto.java

```diff
@@ -0,0 +1,31 @@
+package com.dev.cinema.model.dto;
+
+public class MovieSessionResponseDto {
+    private String showTime;
+    private Long movieId;
+    private Long cinemaHallId;
```

Comment:
```suggestion
    private String showTime;
    private Long movieId;
    private Long cinemaHallId;
    private Long id;
    private String movieTitle;
```

Reply:
@engineer027 previously it seems to be ok as well
the point is that we should return sessionId, showTime as String and some info about Movie and CinemaHall - id or other information